src/main/resources/db/migration/V1.18__add-artefact_archive-to-artefact-mv.sql

```diff
@@ -0,0 +1,37 @@
+--
+-- Add artefact_archived to the Artefact Materialised View so it can return all the artefacts
+--
+DROP MATERIALIZED VIEW IF EXISTS sdp_mat_view_artefact;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS sdp_mat_view_artefact AS
```

Comment:
We can execute the drop before this line so does not need 'IF NOT EXISTS' here

Reply:
good spot. Removed.